spec/models/term_spec.rb

```diff
@@ -89,9 +112,14 @@
       expect(term.uuid).not_to be blank?
     end
 
-    it 'fails if there is a temporary term for the label given' do
-      term
-      expect { FactoryBot.create(:temp_term) }.to raise_error ActiveRecord::RecordInvalid
+    context 'when there is a temporary term for the label given' do
+      let(:term_0) { FactoryBot.create(:temp_term) }
+      let(:term) { FactoryBot.build(:temp_term, vocabulary: term_0.vocabulary) }
+
+      it 'returns validation errors' do
+        expect(term.save).to be false
+        expect(term.errors.full_messages).to include 'Uri has already been taken'
```

Comment:
```suggestion
        expect(term.errors.full_messages).to include 'Uri has already been added to this vocabulary'
```
Same suggested error wording change as above.